rustls/tests/common/mod.rs

```diff
@@ -408,6 +415,7 @@ impl KeyType {
 
     pub fn ca_distinguished_name(&self) -> &'static [u8] {
         match self {
+            // TODO: what's up with the &b"blah"[..] ceremony?
```

Comment:
This gives each branch of the match the same type. (The type of `b"hello"` is `[u8; 5]`, the type of `&b"hello"[..]` is `&[u8]`).

Reply:
Sure, the type of `b"hello"` is `&[u8; 5]`, but it will be coerced to `&[u8]` without the explicit ceremony.

Reply:
Feel free to remove the extra ceremony if it is unnecessary!